### PR TITLE
chore: cut v0.69.0

### PR DESCRIPTION
**Pull request by simonpasquier**
## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [X] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Review by xiu — APPROVED** [review]

**Comment by slashpai** [comment]
Huge and terrific release 🚀

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+## 0.69.0 / 2023-11-03
+
+* [CHANGE] Consider secret references without `optional` value to be mandatory instead of optional for `.spec.additionalScrapeConfigs` configs. #5985
+* [CHANGE] Remove `prometheus_operator_rule_validation_triggered_total`, `prometheus_operator_rule_validation_errors_total`, `prometheus_operator_alertmanager_config_validation_triggered_total` and `prometheus_operator_alertmanager_config_validation_errors_total` metrics which have been deprecated since v0.55.0. The same information is available from the `apiserver_admission_webhook_rejection_count` metric available since Kubernetes v1.16.0. #6024
+* [CHANGE/BUGFIX] Disable HTTP2 connections by default to mitigate CVE-2023-44487. #6028
+* [FEATURE] Add support for EC2 service discovery to the ScrapeConfig CRD. #5902
```
Comment (slashpai) [inline CHANGELOG.md:6]: Can we add this PR also here https://github.com/prometheus-operator/prometheus-operator/pull/6012?

**Review by ArthurSens — APPROVED** [review]
:shipit:

**Comment by simonpasquier** [comment]
Thank for the reviews! I'll wait for monday to merge and tag :)

**Review by nicolastakashi — APPROVED** [review]
🚀 Such a great release

**Comment by simonpasquier** [comment]
Only waiting for https://github.com/prometheus-operator/prometheus-operator/pull/6064 which was reported/fixed today.

**Pull request merged**